test/extensions/transport_sockets/tls/ocsp/ocsp_test.cc

```diff
@@ -189,6 +189,40 @@ TEST_F(Asn1OcspUtilityTest, ParseMethodWrongTagTest) {
   expectThrowOnWrongTag(Asn1OcspUtility::parseResponseStatus);
 }
 
+TEST_F(Asn1OcspUtilityTest, ParseResponseDataBadVersionV2Test) {
+  std::vector<uint8_t> data = {
+      // SEQUENCE
+      0x30,
+      7,
+      // invalid version
+      0xa1,
+      0x02,
+      1,
+      0,
```

Comment:
Again, if I understand the documentation correctly, instead of these values you want, for example: `0xa0, 0x02, 1, 1` which means the context-specific, constructed tag 0 (0xa0), integer type (0x02), length 1 byte (1), value (1).

I don't think there's any need to repeat this test with another version -- it's enough to show that any specific version value that is not 0 will trigger a rejection from Envoy.

Reply:
Sure, I will keep only one test.